go.mod

```diff
@@ -1,3 +1,7 @@
-module github.com/libdns/libdns
+module github.com/artknight/libdns-godaddy
```

Comment:
When publishing to libdns/godaddy, be sure to rename this module to github.com/libdns/godaddy, and update all references in this repo to that module name